pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[tool.poetry]
```

Comment:
This file is not required anymore if we use PyPI.

Reply:
removed